userguide/sharing.rst

```diff
@@ -1299,14 +1300,15 @@ for more details.
    sure to configure **all** SMB shares this way. Reboot the Mac
    client after making this change.
 
-These VFS objects do not appear in the drop-down menu and are dynamically
-configured:
+These VFS objects do not appear in the selection box:
 
 * **recycle:** moves deleted files to the recycle directory instead of
-  deleting them. Controlled by a checkbox in the share configuration.
+  deleting them. Controlled by :guilabel:`Export Recycle Bin`.
 
 * **shadow_copy2:** a more recent implementation of
-  :guilabel:`shadow_copy` with some additional features
+  :guilabel:`shadow_copy` with some additional features. *shadow_copy2*
+  and associated *shadow:* parameters are automatically added to the
```

Comment:
I decided to simplify this and remove reference to "shadow:*". I.e. "and associated parameters"